src/UnitConverter/Unit/Pressure/Millibar.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * This file is part of the jordanbrauer/unit-converter PHP package.
+ *
+ * @copyright 2017 Jordan Brauer <jbrauer.inc@gmail.com>
+ * @license MIT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Pressure;
+
+use UnitConverter\Measure;
+use UnitConverter\Unit\{ AbstractUnit, UnitInterface };
+
+/**
+ * Millibar unit data class.
+ *
+ * @version 1.0.0
+ * @since 1.0.0
+ * @author Teun Willems
+ */
+class Millibar extends PressureUnit
+{
+    protected function configure () : void
+    {
+        $this
+            ->setName("bar")
+
+            ->setSymbol("bar")
```

Comment:
Change the symbol to `"mbar"`.

_Source: [wikipedia page – Bar (unit)](https://en.wikipedia.org/wiki/Bar_(unit))_
![image](https://user-images.githubusercontent.com/18744334/31469522-8422b3fc-aea7-11e7-8303-6d5690aae3b4.png)

We won't use the millibar `mb` notation since `mb` is reserved for the computer data units.